src/bitmessageqt/foldertree.py

```diff
@@ -229,6 +248,7 @@ def data(self, column, role):
         return super(Ui_AddressWidget, self).data(column, role)
 
     def setData(self, column, role, value):
+        """"""
```

Comment:
Save account label (if you edit in the the UI, this will be triggered and will save it to keys.dat)